Documentation/creating-new-package.md

```diff
@@ -0,0 +1,29 @@
+# Creating new .NET IoT package
+
+This is a list of steps which need to happen in order to create new package version of .NET IoT.
+
+## Steps
+
+- regenerate device listing (`dotnet run` on https://github.com/dotnet/iot/tree/master/tools/device-listing and fix all warnings)
+  - this always requires manual review
+- ensure all PRs meant for this release are merged
+- ideally until done do not merge any new PRs (technically not required but will avoid confusion and accidental errors)
+- [Internal] create new package:
+  - Go to https://dev.azure.com/dnceng/internal/_build?definitionId=239
+  - Select "Run Pipeline", select row which says variables and add new one: DotNetFinalVersionKind=release (no quotes anywhere)
+  - Run and wait for it to finish
+  - Once it is done and passes go to the artifacts section of the build, and find an artifact called 'Built packages' and from there download the two stable packages
+  - Validate the package (double check version, check there are no unintentional changes)
+- push package to nuget
```

Comment:
As I in the pattern, maybe good to write what's manual from what's automatic. Is this push something manual or automatic? Where should this happen? Still on the previous step or is this a new step?